auth0/src/main/java/com/auth0/android/authentication/ParameterBuilder.kt

```diff
@@ -166,15 +168,16 @@ public class ParameterBuilder private constructor(parameters: Map<String, String
         public const val AUDIENCE_KEY: String = "audience"
 
         /**
-         * Creates a new instance of the builder using default values for login request, e.g. 'openid' for scope.
+         * Creates a new instance of the builder using default values for login request, e.g. 'openid profile email' for scope.
          *
          * @return a new builder
          */
         @JvmStatic
         public fun newAuthenticationBuilder(): ParameterBuilder {
             return newBuilder()
-                .setScope(SCOPE_OPENID)
+                .setScope(OidcUtils.DEFAULT_SCOPE)
```

Comment:
this method will always add as default the scope of "openid profile email". It can be changed later by the dev.